### PR TITLE
8 tiller annotation

### PR DESCRIPTION
**Pull request by ochoaj**

### CODE REVIEW

File: annotations/tiller.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  ![tiller from Gokstad ship](../images/Tiller_Gokstad.jpg)  
+  Elaborately decorated and carved tiller from the Gokstad ship (Nicolaysen Pl. XI, Fig. 1)
+
+</div>
+
+  The tiller was the primary means for steering a Viking ship; by pushing and pulling the tiller fore and aft the attached side rudder rotated laterally in the water. The ON verb _stýra_ literally means holding the tiller and directing the ship but is also used metaphorically in the corpus of Skaldic poetry to refer to a captain's command of one or more ships (Jesch 174). The importance of the tiller and helmsman position is indicated by the elaborate decoration of the Gokstad find, which was carved and painted in three colors, in contrast to the less decorated other parts of the ship (Nicolaysen 67-8).
+
+---
+
+  Jesch, Judith. _Ships and Men in the Late Viking Age: The Vocabulary of Runic Inscriptions and Skaldic Verse._ NED-New edition. Woodbridge, Suffolk, UK ; Rochester, NY:
```
Comment (mehtas27) [inline annotations/tiller.md:19]:
```suggestion
 Jesch, Judith. _Ships and Men in the Late Viking Age: The Vocabulary of Runic Inscriptions and Skaldic Verse._ NED-New edition. Woodbridge, Suffolk, UK ; Rochester, NY: 
```

File: annotations/tiller.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+  The tiller was the primary means for steering a Viking ship; by pushing and pulling the tiller fore and aft the attached side rudder rotated laterally in the water. The ON verb _stýra_ literally means holding the tiller and directing the ship but is also used metaphorically in the corpus of Skaldic poetry to refer to a captain's command of one or more ships (Jesch 174). The importance of the tiller and helmsman position is indicated by the elaborate decoration of the Gokstad find, which was carved and painted in three colors, in contrast to the less decorated other parts of the ship (Nicolaysen 67-8).
+
+---
+
+  Jesch, Judith. _Ships and Men in the Late Viking Age: The Vocabulary of Runic Inscriptions and Skaldic Verse._ NED-New edition. Woodbridge, Suffolk, UK ; Rochester, NY: 
+Boydell & Brewer, 2001. https://www.jstor.org/stable/10.7722/j.ctt163tb4f.
+
+
+  Nicolaysen, N. _Langskibet fra Gokstad ved Sandefjord_. Kristiania: Alb. Cammermeyer, 1882.
```
Comment (mehtas27) [inline annotations/tiller.md:23]:
```suggestion
 Nicolaysen, N. _Langskibet fra Gokstad ved Sandefjord_. Kristiania: Alb. Cammermeyer, 1882.
```

Comment (mehtas27) [inline annotations/tiller.md:23]:
```suggestion
   Nicolaysen, N. _Langskibet fra Gokstad ved Sandefjord_. Kristiania: Alb. Cammermeyer, 1882.
```

File: annotations/tiller.md (hunk 1 above)
Comment (mehtas27) [inline annotations/tiller.md:19]:
```suggestion
   Jesch, Judith. _Ships and Men in the Late Viking Age: The Vocabulary of Runic Inscriptions and Skaldic Verse._ NED-New edition. Woodbridge, Suffolk, UK ; Rochester, NY: 
```

**Review by mehtas27 — CHANGES_REQUESTED** [review]

File: annotations/tiller.md (hunk 2 above)
Comment (apjmason) [reply annotations/tiller.md:23]: Spaces removed and readded

**Review by apjmason — APPROVED** [review]
Looks good!

**Review by mehtas27 — APPROVED** [review]

**Pull request merged**